topic_manager.go

```diff
@@ -358,8 +358,8 @@ func (tmc *TopicManagerConfig) tableCleanupPolicy() string {
 // initialize the config with reasonable defaults
 func NewTopicManagerConfig() *TopicManagerConfig {
 	cfg := new(TopicManagerConfig)
-	cfg.Table.Replication = 2
-	cfg.Stream.Replication = 2
+	cfg.Table.Replication = 1
```

Comment:
Actually I'd say replication factor 2 by default makes more sense, so why not update the comment instead? Also since this is has been the default configuration for a longer time, wouldn't it be kind of a breaking change and force everyone to update their config just to make sure to continue using replication factor of 2?

Reply:
make sense.